CHANGELOG.md

```diff
@@ -11,6 +11,79 @@ All notable changes to Botonic will be documented in this file.
   </summary>
 </details>
 
+## [0.14.0] - 2020-30-09
+
+### Added
+
+- [@botonic/cli](https://www.npmjs.com/package/@botonic/cli)
+
+  - New bot template `dynamo` using Botonic dynamoDB plugin from a JS bot.
+
+* [@botonic/react](https://www.npmjs.com/package/@botonic/react)
+
+  - Improved behavior when end user messages are lost due to connection issues. Missed inputs will be resent when connection is regained.
+  - End user messages will be displayed with less opacity until they are correctly delivered to the server.
+  - Added classNames to blob messages by default, under the following format:
+    `{type}-{from} {customTypeName}`
+    e.g.: Text messages from user -> `text-user`
+    e.g.: Custom messages from bot -> `custom-bot my-custom-message-name`
+  - Passing `mobileStyle` in `webchat/index.js`is now allowed.
+  - Added arrow buttons for better interaction with Webchat Carousels. They can be customized.
+  - Disabling timestamps in the custom Messages using the `enableTimestamps` prop is now allowed.
+  - Defined where to store the botonicState using the variable storage.
+  - The methods `openCoverComponent`, `closeCoverComponent` and `toggleCoverComponent` are available in the browser, through the Botonic object, so that the `coverComponent` can be shown/hidden manually.
+
+- [@botonic/plugin-contentful](https://www.npmjs.com/package/@botonic/plugin-contentful)
+
+  - Decorator for Contentful that can distribute the delivery requests amongst different spaces/environments.
+  - Added a new tool to modify Contentful spaces exported as json file with "contentful space export".
```

Comment:
"modify Contentful spaces" => "modify the locales of Contentful spaces"
Append: " The tool is able to change the name of a locale without altering the contents and to remove unwanted locales